build/pkgs/debugpy/spkg-configure.m4

```diff
@@ -1 +1,14 @@
-SAGE_SPKG_CONFIGURE([debugpy], [SAGE_PYTHON_PACKAGE_CHECK([debugpy])])
+SAGE_SPKG_CONFIGURE([debugpy], [
+  SAGE_PYTHON_PACKAGE_CHECK([debugpy])
+], [
+  dnl REQUIRED-CHECK
+  dnl
+  dnl Skip debugpy if ipykernel from the system will be used.
+  dnl This allows downstream packagers to treat debugpy (a
+  dnl somewhat problematic package) as optional.
```

Comment:
include a link to a discussion what makes it problematic?

Reply:
There isn't much of a discussion, only a comment in `package.mask` that says "Very fragile, keeps breaking." But IIRC it's from all the vendored junk under https://github.com/microsoft/debugpy/tree/main/src/debugpy/_vendored/pydevd